target/linux/ath79/dts/qca9563_zte_mf286.dtsi

```diff
@@ -0,0 +1,156 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+// Copyright (c) 2021 Cezary Jackiewicz
+// Copyright (c) 2021, 2022 Lech Perczak
+#include "qca956x.dtsi"
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+#include <dt-bindings/leds/common.h>
+
+/ {
+	aliases {
+		led-boot = &led_debug;
+		led-failsafe = &led_debug;
+		led-running = &led_debug;
+		led-upgrade = &led_debug;
+		label-mac-device = &eth0;
+	};
+
+	leds {
+		compatible = "gpio-leds";
+
+		/* Hidden green SMD LED below signal strength LEDs
+		 * Visible through slits underside of the case,
+		 * and slightly through the case below signal state LEDs */
+		led_debug: led-0 {
+			function = LED_FUNCTION_DEBUG;
+			gpios = <&gpio 7 GPIO_ACTIVE_LOW>;
+			default-state = "on";
+		};
+	};
+
+	keys {
+		compatible = "gpio-keys";
+
+		reset {
+			label = "reset";
+			linux,code = <KEY_RESTART>;
+			gpios = <&gpio 2 GPIO_ACTIVE_LOW>;
+			debounce-interval = <60>;
+		};
+
+		wps {
+			label = "wps";
+			linux,code = <KEY_WPS_BUTTON>;
+			gpios = <&gpio 1 GPIO_ACTIVE_LOW>;
+			debounce-interval = <60>;
+		};
+
+		wifi {
+			label = "wifi";
+			linux,code = <KEY_RFKILL>;
+			gpios = <&gpio 8 GPIO_ACTIVE_LOW>;
+			debounce-interval = <60>;
+		};
+	};
+
+	/* This GPIO is used to reset whole board _including_ the modem */
+	gpio-restart {
+		compatible = "gpio-restart";
+		gpios = <&gpio 5 GPIO_ACTIVE_HIGH>;
+		active-delay = <3000>;
+		inactive-delay = <1000>;
+	};
+};
+
+&gpio {
+	/* GPIO19 is used as a mask to enable WLAN LED
+	 * in stock firmware, which is controlled directly
+	 * by 5GHz Wi-Fi chip, which currently is inactive
+	 * in OpenWrt */
```

Comment:
Just noticed this here... but unfortunately not earlier.

So, upstream [linux has some comments](https://www.kernel.org/doc/html/v5.10/process/coding-style.html#commenting) on how to consistently format the multi line comments.

They like it that way:

         /*
          * comment line 1
          * comment line 2
          */

(unless it's `net` code)

Reply:
Fixed.